src/Zikula/CoreBundle/Maker/ExtensionMaker.php

```diff
@@ -0,0 +1,195 @@
+
+        return [
+            ['name' => $bundleClassName, 'prefix' => '', 'suffix' => $type, 'template' => 'BundleClass.tpl.php'],
+            ['name' => $bundleClassName, 'prefix' => 'DependencyInjection', 'suffix' => 'Extension', 'template' => 'DIExtensionClass.tpl.php'],
+            ['name' => $extensionName . $type, 'prefix' => '', 'suffix' => 'Installer', 'template' => 'InstallerClass.tpl.php'],
+        ];
+    }
+
+    private function generateClasses(string $namespace, string $type): string
+    {
+        $bundleClassFullName = '';
+        foreach ($this->getClassesToGenerate($namespace, $type) as $classInfo) {
+            $bundleClassNameDetails = $this->localGenerator->createClassNameDetails(
+                $classInfo['name'],
+                $classInfo['prefix'],
+                $classInfo['suffix'],
+                'Invalid!' . $classInfo['name']
+            );
+            $bundleClassFullName = ('' === $classInfo['prefix'] && $type === $classInfo['suffix']) ? $bundleClassNameDetails->getShortName() : $bundleClassFullName;
+            $this->localGenerator->generateClass(
+                $bundleClassNameDetails->getFullName(),
+                dirname(__DIR__) . '/Resources/skeleton/extension/' . $classInfo['template'],
+                [
+                    'namespace' => $namespace . $type,
+                    'type' => $type,
+                    'name' => $bundleClassNameDetails->getShortName(),
+                    'vendor' => mb_substr($namespace, 0, mb_strpos($namespace, '\\'))
+                ]
+            );
+        }
+
+        return $bundleClassFullName;
+    }
+
+    private function getFilesToGenerate(): iterable
+    {
+        return [
+            'Resources/config/services.yaml' => 'services.yaml.tpl.php',
+            'README.md' => 'README.md.tpl.php',
+            'composer.json' => 'composer.json.tpl.php',
+            'LICENSE.txt' => 'MIT.txt.tpl.php',
+        ];
+    }
+
+    private function generateFiles(string $namespace, string $type, string $bundleClass): void
+    {
+        foreach ($this->getFilesToGenerate() as $targetPath => $templateName) {
+            $this->localGenerator->generateFile(
+                $this->extensionPath . '/' . $targetPath,
+                dirname(__DIR__) . '/Resources/skeleton/extension/' . $templateName,
+                [
+                    'namespace' => $namespace . $type,
+                    'type' => $type,
+                    'vendor' => mb_substr($namespace, 0, mb_strpos($namespace, '\\')),
+                    'name' => mb_substr($namespace, mb_strpos($namespace, '\\') + 1),
+                    'bundleClass' => $bundleClass
+                ]
+            );
+        }
+    }
+
+    private function generateBlankFiles(): void
+    {
+        $fs = new Filesystem();
+        $fs->mkdir($this->extensionPath . '/Resources/doc');
+        $fs->touch($this->extensionPath . '/Resources/doc/index.md');
```

Comment:
Please add `/Resources/doc/help/en/README.md` for the help system introduced in Zikula 3.